backend/grant-service/src/main/kotlin/grant/services/Applications.kt

```diff
@@ -502,6 +502,36 @@ class ApplicationService(
         )
     }
 
+    suspend fun transferApplication(
+        ctx: DBContext,
+        actor: Actor,
+        currentProject: String?,
+        applicationId: Long,
+        transferToProjectId: String
+    ) {
+        if (currentProject.isNullOrEmpty()) {
+            RPCException.fromStatusCode(HttpStatusCode.BadRequest, "Not in a project. No applications in Personal workspace")
```

Comment:
Missing `throw` keyword
```suggestion
            throw RPCException.fromStatusCode(HttpStatusCode.BadRequest, "Not in a project. No applications in Personal workspace")
```